src/hepattn/models/maskformer.py

```diff
@@ -61,10 +61,13 @@ def __init__(
         self.input_sort_field = input_sort_field
         self.use_attn_masks = use_attn_masks
         self.use_query_masks = use_query_masks
+        self.log_attn_mask = log_attn_mask
+        self.step_ = 0
```

Comment:
btw I think normally leading underscores are used to indicate internal use, not trailing.